### PR TITLE
Updated format of generating BUFFER_QUEUE in buffers_defaults templates

### PR DESCRIPTION
**Pull request by OleksandrKozodoi**
Signed-off-by: Oleksandr Kozodoi <oleksandrx.kozodoi@intel.com>

This PR includes necessary changes for correct generating BUFFER_QUEUE values in DB. Changes are based on the [schema.md](https://github.com/Azure/sonic-swss/blob/master/doc/swss-schema.md#buffer_queue_table) 
#### Why I did it
Change format of generating BUFFER_QUEUE in DB according to [schema.md](https://github.com/Azure/sonic-swss/blob/master/doc/swss-schema.md#buffer_queue_table) and [yang-model](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-buffer-queue.yang).
##### Old format:
```
    "BUFFER_QUEUE": {
        "Ethernet0,Ethernet100,Ethernet104,Ethernet108,Ethernet112,Ethernet116,Ethernet12,Ethernet120,Ethernet124,Ethernet16,Ethernet20,Ethernet24,Ethernet28,Ethernet32,Ethernet36,Ethernet4,Ethernet40,Ethernet44,Ethernet48,Ethernet52,Ethernet56,Ethernet60,Ethernet64,Ethernet68,Ethernet72,Ethernet76,Ethernet8,Ethernet80,Ethernet84,Ethernet88,Ethernet92,Ethernet96|queue": {
            "profile": "profile"
        },
        "Ethernet0,Ethernet100,Ethernet104,Ethernet108,Ethernet112,Ethernet116,Ethernet12,Ethernet120,Ethernet124,Ethernet16,Ethernet20,Ethernet24,Ethernet28,Ethernet32,Ethernet36,Ethernet4,Ethernet40,Ethernet44,Ethernet48,Ethernet52,Ethernet56,Ethernet60,Ethernet64,Ethernet68,Ethernet72,Ethernet76,Ethernet8,Ethernet80,Ethernet84,Ethernet88,Ethernet92,Ethernet96|queue": {
            "profile": "profile"
        }
    },
```
##### New format:
```
    "BUFFER_QUEUE": {
        "Ethernet0|queue": {
            "profile": "profile"
        },
        "Ethernet0|queue": {
            "profile": "profile"
        },
        "Ethernet4|queue": {
            "profile": "profile"
        },
        "Ethernet4|queue": {
            "profile": "profile"
        },
        "Ethernet8|queue": {
            "profile": "profile"
        },
        "Ethernet8|queue": {
            "profile": "profile"
        },
        ...
    }
```
#### How I did it
Updated structure of buffers_defaults jinja templates.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

**Comment by OleksandrKozodoi** [comment]
@yxieca @lguohan @neethajohn Can you take a look? Thank you in advance.

**Comment by yxieca** [comment]
@OleksandrKozodoi does this change affect warm reboot? If we wb from a version with old format to this new version with new format. will reconciliation logic judge the configuration as different?

**Comment by yxieca** [comment]
@vaibhavhd for awareness

**Comment by OleksandrKozodoi** [comment]
@yxieca This change doesn't affect warm reboot. Thanks for your comment.

File: device/barefoot/x86_64-accton_wedge100bf_32x-r0/montara/buffers_defaults_t0.j2 [hunk 1]
```diff
@@ -63,21 +63,17 @@
     },
 {%- endmacro %}
 
-{%- macro generate_pg_profils(port_names) %}
```
Comment (neethajohn) [inline device/barefoot/x86_64-accton_wedge100bf_32x-r0/montara/buffers_defaults_t0.j2:66]: Why is this removed? 3-4 should use the pg_lossless profiles?

Comment (OleksandrKozodoi) [reply device/barefoot/x86_64-accton_wedge100bf_32x-r0/montara/buffers_defaults_t0.j2:66]: There is the correct template in [buffers_config.j2](https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/buffers_config.j2#L160-L178). If we remove this statement from buffers_defaults_t0.j2, buffers_config.j2 won't import macro generate_pg_profils from buffers_defaults_t0.j2 and use the correct template of BUFFER_PG.

**Review by neethajohn — APPROVED** [review]
